pages/en/lb4/Schemas.md

```diff
@@ -9,9 +9,127 @@ summary:
 ---
 {% include content/tbd.html %}
 
-> Stub page per [#554](https://github.com/strongloop/loopback-next/issues/554).
+<!-- TOC -->
```

Comment:
The doc site automatically creates a TOC for each page (unless you specify not to).  So, this TOC is redundant.